sygnal/apnstruncate.py

```diff
@@ -15,10 +15,10 @@
 # Copied and adapted from
 # https://raw.githubusercontent.com/matrix-org/pushbaby/master/pushbaby/truncate.py
 import json
-from typing import List, Tuple, Union
+from typing import Any, Dict, List, Tuple, Union
 
 
-def json_encode(payload):
+def json_encode(payload: Dict[str, Any]) -> bytes:
```

Comment:
`payload` doesn't have to be a dict, it can be anything that can be translated into json (so pretty much any native type). I think the right type for it would be `Any`.

Reply:
Now payload should look like this `(payload: Any) -> bytes` or it should also return `Any `

Reply:
`bytes` is the right value, thanks for changing the payload type :)